.eslintrc.json

```diff
@@ -0,0 +1,3 @@
+{
+  "extends": "eslint-config-edx"
+}
```

Comment:
I know we'd discussed that there were going to be some overrides but I don't remember what they were now. It makes sense to check this in as you have it, and then we can add overrides if or when we need them in follow-on PRs.